src/main/webapp/i18n/de/user-management.json

```diff
@@ -102,7 +102,8 @@
             "search": "Suche",
             "loading": "Laden...",
             "searchError": "Der Suchbegriff muss leer oder mindestens 3 Zeichen lang sein.",
-            "jenkinsChange": "Die Änderung des Benutzernamens wird für dein aktuelles Setup (Jenkins) nicht unterstützt. Möglicherweise musst du den Benutzernamen in Jenkins manuell ändern.\nDu hast den Benutzer <i>{{ oldLogin }}</i> auf <i>{{ newLogin }}</i> aktualisiert."
+            "jenkinsChange": "Die Änderung des Benutzernamens wird für dein aktuelles Setup (Jenkins) nicht unterstützt. Möglicherweise musst du den Benutzernamen in Jenkins manuell ändern.\nDu hast den Benutzer <i>{{ oldLogin }}</i> auf <i>{{ newLogin }}</i> aktualisiert.",
```

Comment:
Out of curiosity regarding `Du hast den Benutzer <i>{{ oldLogin }}</i> auf <i>{{ newLogin }}</i> aktualisiert.`.

Is it the Artemis user that has been changed? I'd probably add that info.

Reply:
I am actually unsure which user is meant there 😅 I did not add this line of text, i just added a comma to that line to add another string.

Reply:
It's updating the Artemis-User, but not the Jenkins-User. See UserService#updateUserInConnectorsAndAuthProvider.